src/common.cpp

```diff
@@ -64,7 +64,7 @@ std::string sockaddr_to_hostport(const struct sockaddr *addr)
     if (addr->sa_family == AF_INET6) {
         return "[" + std::string(hbuf) + "]:" + std::string(pbuf);
     } else if (addr->sa_family == AF_UNIX) {
-        return std::string(pbuf) + " [UNIX]";
+        return std::string(addr->sa_data) + " [AF_UNIX]";
```

Comment:
can you show output

Reply:
in linux it's enough to give the `pbuf` because `getnameinfo` knows how to fill it with the address and the output is good. 

In windows, I changed to more common way that works both for linux and windows with `addr->sa_data` which is the address directly. 
Also changed `UNIX` to `AF_UNIX`  - more generic
BTW, now I see another place to change.. 

![af unix print](https://user-images.githubusercontent.com/44922030/183435562-5b9aefff-6b49-4767-973d-9f07524bb5f8.JPG)

Reply:
Can we adapt https://github.com/Mellanox/sockperf/blob/sockperf_v2/src/iohandlers.cpp#L41
to
`printf("[%2d] ADDR = %s # %s\n", list_count++, pbuf, PRINT_PROTOCOL(data->sock_type));`

Reply:
just one thing, it's SOCK_STREAM / SOCK_DGRAM rather then TCP / UDP 
so output like 
`[ 0] ADDR = \tmp\test # SOCK_STREAM`

Reply:
> just one thing, it's SOCK_STREAM / SOCK_DGRAM rather then TCP / UDP so output like `[ 0] ADDR = \tmp\test # SOCK_STREAM`

TCP/UDP is clear for end user and exist. I do not see benefits in SOCK_STREAM / SOCK_DGRAM

Reply:
it's just a semantic we agreed in UDS patch, that AF_UNIX is not a TCP/UDP protocol and it might be confusing. 
`AF_INET + SOCK_STREAM  = TCP` 
`AF_UNIX + SOCK_STREAM !=TCP`

That's why in the print I use `PRINT_SOCKET_TYPE` and not `PRINT_PROTOCOL`

Reply:
The behavior or AF_UNIX + SOCK_STREAM sockets is comparable to TCP.
So seeing
`[ 0] ADDR = \tmp\test # TCP` 
in output points to AF_UNIX + SOCK_STREAM
and
`[ 0] ADDR = 1.1.1.1 # TCP` 
points to AF_INET + SOCK_STREAM
Let keep identical output format for inet and unix